### ISSUE TITLE
Diff coverage is not correct

### ISSUE THREAD

**Issue by eungjun-yi** [naver/cover-checker#12]
The result of coverage check is always `0/0 (0%)` even if the pull request has made some modification and some lines are not covered by tests.

What I Did:
```
java -jar target/coverchecker-1.3.3-jar-with-dependencies.jar -c /home/eungjun/mysrc/diff-cov-sample/build/reports/jacoco/test/jacocoTestReport.xml -r eungjun-yi/diff-cov-sample -p 4 -g <my-personal-access-token> -t 1
```

What I Expected:

```
1/2 (50.0%)
```

Actual Result:
![image](https://user-images.githubusercontent.com/1129852/68572385-95a72f00-04a8-11ea-8edc-014c3003cd99.png)

Here is the correct report by codecov on [the pull request](https://github.com/eungjun-yi/diff-cov-sample/pull/4).

![image](https://user-images.githubusercontent.com/1129852/68572709-78bf2b80-04a9-11ea-927f-fb78da7d38fc.png)

**Comment by ksmail13** [comment]
Hello, thanks for your report.

This issue occurs when to combine diff and coverage reports.
`NewCoverageChecker` checks only java file like [this](https://github.com/naver/cover-checker/blob/master/src/main/java/com/naver/nid/cover/checker/NewCoverageChecker.java#L50).
Because, When I design this project, I considered only java.

I will fix ASAP.

Thanks.

**Comment by ksmail13** [comment]
@eungjun-yi 
I fix this issue in [1.3.4](https://github.com/naver/cover-checker/releases/tag/1.3.4)
Would you try again?

**Comment by ksmail13** [comment]
I close the issue.
Please reopen when you get same issue.
